org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/customservice/synapse/mediator/tryout/debugger/BreakpointEventProcessor.java

```diff
@@ -94,7 +100,7 @@ private String getProperty(String context) {
         property.addProperty(TryOutConstants.CONTEXT, context);
 
         try {
-            return commandClient.sendCommand(property.toString());
+            return commandClient.sendCommandWithRetry(property.toString(), 2);
```

Comment:
What is the purpose of making maxRetries an argument? Based on the scenario are we changing the number of reattempts?

Reply:
Currently it is not used. I just made it as an argument, in case if we need it in future.

Reply:
Fixed in [e9b9e3c](https://github.com/wso2/mi-language-server/pull/430/commits/e9b9e3c2815dd6d57c4ea2ed36da3460bb2d7104)